cfgov/jinja2/v1/ask-cfpb/landing-page.html

```diff
@@ -37,48 +37,47 @@ <h2 class="u-show-on-mobile u-mt30">
                 {% endfor %}
             </ul>
 
-            <div class="question-categories">
-                <div class="u-hide-on-mobile">
-
-                    <div class="content-l content-l__equal-height">
-
-                        {% for portal in portal_cards %}
-                            <div class="content-l_col content-l_col-1-2 content-l_col__equal-height">
-                                <article class="m-card content-l_body__equal-height">
-                                    <h2>
-                                        <a href="{{ portal.url }}">
-                                        <span>{{ category_icon.render( portal.topic ) }}</span>
-                                        <span>{{ portal.title }}</span>
-                                        </a>
-                                    </h2>
-
-                                    <ul class="m-list question_list m-list__flush-left u-mt10 u-mb30">
-                                        {% for page in portal.featured_answers %}
-                                            <li class="m-list_item">
-                                            {{ partially_styled_link.render({
-                                                'plain_text': page.question,
-                                                'underlined_text': _('Read answer'),
-                                                'url': page.url })
-                                            }}
-                                            </li>
-                                        {% endfor %}
-                                    </ul>
-
-                                    <div class="content-l_footer__equal-height">
-                                        <p class="short-desc u-mb0">
-                                            <a href="{{ portal.url }}">
-                                               {{ _('Learn more about') }} {{ portal.title|lower }}
-                                            </a>
-                                        </p>
-                                    </div>
-
-                                </article>
+            <div class="u-hide-on-mobile question-categories">
```

Comment:
One thing I'm wondering (probably outside the scope of this PR) is whether, instead of being swapped out for a list of links, the Ask cards could reduce to their header links on mobile? We could probably use flex & `row-reverse` to flip the icons to the right and match the existing links' layout...

Reply:
I wondered this too. This should probably be hashed out in https://github.com/cfpb/design-system/ in the future.